dev/ci/docker_linux/Dockerfile

```diff
@@ -27,7 +27,8 @@ RUN apt-get install -y --no-install-recommends \
   unzip \
   apt-transport-https \
   ca-certificates \
-  gnupg
+  gnupg \
+  procps
```

Comment:
@christopherfujino  - should I split this into a separate PR?

This is needed so that `kill` is available in the container.